subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/GradlePluginDevelopmentExtension.java

```diff
@@ -152,15 +152,21 @@ public void plugins(Action<? super NamedDomainObjectContainer<PluginDeclaration>
     /**
      * Whether the plugin should automatically configure the publications for the plugins.
      * @return true if publishing should be automated, false otherwise
+     *
+     * @deprecated non-automatic publishing of plugins will no longer be supported
      */
+    @Deprecated
     public boolean isAutomatedPublishing() {
         return automatedPublishing;
     }
 
     /**
      * Configures whether the plugin should automatically configure the publications for the plugins.
      * @param automatedPublishing whether to automated publication
+     *
+     * @deprecated non-automatic publishing of plugins will no longer be supported
      */
+    @Deprecated
```

Comment:
Invoking this method should print a deprecation warning as well. The text should also say that it is effectively ignored when using the plugin publish >= 1.0